frb_cors/pubspec.yaml

```diff
@@ -0,0 +1,12 @@
+name: frb_cors
```

Comment:
Shall we publish it to pub.dev?

If so, well, to be consistent maybe we should write down `flutter_rust_bridge_something`?

And need some modification to justfile as well...